core/CliMulti/Process.php

```diff
@@ -236,4 +240,18 @@ private static function isProcFSMounted()
         return strpos($type, 'proc') === 0;
     }
 
+    /**
+     * @return int[] The ids of the currently running processes
+     */
+     static function getRunningProcesses()
+    {
+        $ids = explode("\n", trim(`ps ex 2>/dev/null | awk '{print $1}'`));
```

Comment:
this will now be executed before `self::commandExists('ps')` - i'm wondering how it fails when `ps` is not defined on the system. to simulate case where the command does not exist, can you eg.  replace `ps` by `psxxx` and check that there is no error displayed on screen?